tests/unit/helpers.spec.js

```diff
@@ -51,7 +51,7 @@ describe('helper functions', () => {
       }
     }]
 
-    assignKeyHandler(e, keyMap, {})
+    assignKeyHandler(e, keyMap, {}, [])
```

Comment:
Don't the last argument if you set it to default in function definition. Remove for all case where array is empty